src/main.py

```diff
@@ -0,0 +1,72 @@
+import asyncio
+from typing import List
+
+import httpx
+
+from model.constants import max_candidates_num
+from model.developer import Developer
+from model.fetcher import fetch_stargazers_for_repo
+from model.repository import Repository
+from tqdm import tqdm
```

Comment:
Let's follow import order suggested by pep8
* std lib
* external libs
* imports from the current module